src/Avalonia.X11/X11Window.cs

```diff
@@ -98,14 +98,26 @@ public X11Window(AvaloniaX11Platform platform, IWindowImpl popupParent)
                 valueMask |= SetWindowValuemask.ColorMap;   
             }
 
-            _handle = XCreateWindow(_x11.Display, _x11.RootWindow, 10, 10, 300, 200, 0,
+            int defaultWidth = 300, defaultHeight = 200;
+
+            if (!_popup)
+            {
+                var monitor = Screen.AllScreens.OrderBy(x => x.PixelDensity)
```

Comment:
At this point Position property is always (0,0). The window isn't even created yet.